Packs/jamf/Integrations/JamfProtectEventCollector/README.md

```diff
@@ -0,0 +1,45 @@
+Use this integration to fetch Audits and Alerts from Jamf Protect as events in XSIAM.
```

Comment:
what about the API version?

Reply:
Couldn't find the right version and decided to leave it blank.